### PR TITLE
Adds popout cake!

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: Traitors and Nukeops can now buy the Pop-out cake in their uplinks -- a cake so big, it can hold a person inside!
/:cl:

Closet ops 2.0!

### CODE REVIEW

File: code/modules/food_and_drinks/food/popout_cake.dm [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		visible_message("<span class='notice'>[user] cuts off the last slice of \the [src].</span>")
+		qdel(src)
+
+/obj/item/reagent_containers/food/snacks/store/cake/popout/MouseDrop_T(atom/movable/O, mob/living/user)
+	if(!QDELETED(thing_inside) && istype(thing_inside))
+		return
+	if(O == user)
+		visible_message("<span class='notice'>[user] begins climbing into \the [src]</span>")
+	else
+		visible_message("<span class='notice'>[user] begins putting \the [O] into \the [src]</span>")
```
Comment (Fox-McCloud) [inline code/modules/food_and_drinks/food/popout_cake.dm]: `\the` is redundant....all you need is `[O]`

**Comment by calzilla1** [comment]
15 is way to fucking much

**Comment by praisenarsie** [comment]
this is ported from somewhere, p sure its vg or para

**Comment by nicbn** [comment]
15 to have a stealth item which sprite is used only by the item itself

**Comment by Fox-McCloud** [comment]
Not Paradise.

**Comment by MoreRobustThanYou** [comment]
@praisenarsie vg

**Comment by DaxDupont** [comment]
Why is this tator only, it doesn't really have much of an advantage.
Let the chef be able to make these!

**Pull request closed without merging**